ruby-saml.gemspec

```diff
@@ -25,28 +25,33 @@ Gem::Specification.new do |s|
   s.summary = %q{SAML Ruby Tookit}
   s.test_files = `git ls-files test/*`.split("\n")
 
-  s.add_runtime_dependency('uuid', ['~> 2.3'])
-  s.add_runtime_dependency('nokogiri', ['~> 1.5.0'])
+  s.add_runtime_dependency('uuid', '~> 2.3')
+  if RUBY_VERSION < '1.9'
+    # 1.8.7
+    s.add_runtime_dependency('nokogiri', '~> 1.5.10')
+  else
+    s.add_runtime_dependency('nokogiri', '~> 1.6')
```

Comment:
@phlipper I will make the change to `~> 1.6.0`, but unfortunately setting it to `~> 1.5` will still cause Ruby 1.8 to try to install Nokogiri 1.6.x, despite it not being compatible :(